authority/provisioners_test.go

```diff
@@ -42,19 +38,19 @@ func TestGetEncryptedKey(t *testing.T) {
 					http.StatusNotFound, context{}},
 			}
 		},
-		"fail-invalid-type-found": func(t *testing.T) *ek {
-			c, err := LoadConfiguration("../ca/testdata/ca.json")
-			assert.FatalError(t, err)
-			a, err := New(c)
-			assert.FatalError(t, err)
-			a.encryptedKeyIndex.Store("foo", 5)
-			return &ek{
-				a:   a,
-				kid: "foo",
-				err: &apiError{errors.Errorf("stored value is not a string"),
-					http.StatusInternalServerError, context{}},
-			}
-		},
+		// "fail-invalid-type-found": func(t *testing.T) *ek {
```

Comment:
still need?

Reply:
No, you cannot do that now, everything is integrated into the Collection type that enforces a provisioner.Interface in the Store method.